tests/ImageSharp.Tests/Processing/Processors/Transforms/RotateTests.cs

```diff
@@ -26,6 +29,26 @@ public class RotateTests : FileTestBase
             RotateType.Rotate270
         };
 
+        public static readonly TheoryData<string> ResamplerNames
+            = new TheoryData<string>
+        {
+            nameof(KnownResamplers.Bicubic),
```

Comment:
Now that we introduced `TransformProcessor` and `TransformTests`, I think `RotateTests` should focus mostly on corner cases it specializes. (See my comment on `TransformTests`)